### ISSUE TITLE
Ads.txt Problem

### ISSUE THREAD

**Issue by JoshTango** [OrchardCMS/OrchardCore#5671]
Google AdSense requires me to have ads.txt file in my root folder.
How do I do this with Orchard Core?

**Comment by deanmarcussen** [comment]
You would put the `ads.txt` in your main startup projects `wwwroot` folder and as long as you have `.UseStaticFiles()` called in your main startup it will be available

**Comment by JoshTango** [comment]
yes for developers, but what about for non developers.
I am a developer but I need non developers to be able to do this.

**Comment by deanmarcussen** [comment]
You should write some code to serve the content you want then.

Here's an example of how I manage `robots.txt` via the admin.
https://github.com/ThisNetWorks/ThisNetWorks.OrchardCore.Seo/tree/develop/src/ThisNetWorks.OrchardCore.Seo.Robots

**Comment by JoshTango** [comment]
yes i know i can do this, maybe I should have said "feature request" in the title.
I want this functionality out of the box in Orchard Core

**Comment by sebastienros** [comment]
A solution is also covered by this issue: https://github.com/OrchardCMS/OrchardCore/issues/5610

Another solution is to have a custom Ads.txt feature part of an SEO module that would provide a custom UI to manage it.
